external/fv3fit/fv3fit/sklearn/_wrapper.py

```diff
@@ -32,6 +34,66 @@ def _tuple_to_multiindex(d: tuple) -> pd.MultiIndex:
     return pd.MultiIndex.from_tuples(list_, names=names)
 
 
+@_shared.io.register("sklearn")
+@register_estimator("sklearn", RandomForestHyperparameters)
+@register_estimator("rf", RandomForestHyperparameters)
+@register_estimator("random_forest", RandomForestHyperparameters)
+@register_estimator("sklearn_random_forest", RandomForestHyperparameters)
+class RandomForest(Estimator):
```

Comment:
Nice. I like this way of providing a clean interface to the overly complex sklearn code. It might be worth renaming this module to `_random_forest.py` and making `RegressorEnsemble` and `SklearnWrapper` private APIs.

Reply:
Also, this has exposed that offline_ml_diags/_plot_input_sensitivity.py is tightly coupled to fv3fit's internals, because it type hints specific model types rather than having a jacobian/feature importances method defined. I'll work on that in another PR.